### PR TITLE
Basic integration tests for NodeServices

### PR DESCRIPTION
**Pull request by SteveSandersonMS**
Integration tests for RPC from .NET to Node.

### CODE REVIEW

**Comment by SteveSandersonMS** [comment]
cc @mkArtakMSFT

File: src/Middleware/NodeServices/test/NodeServicesTest.cs [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        }
+
+        [Fact]
+        public async Task CanPassParametersWithCamelCaseNameConversion()
+        {
+            // Act
+            var result = await _nodeServices.InvokeExportAsync<string>(
+                ModulePath("testCases"),
+                "echoComplexParameters",
+                new ComplexModel { StringProp = "Abc", IntProp = 123, BoolProp = true });
```
Comment (ryanbrandenburg) [inline src/Middleware/NodeServices/test/NodeServicesTest.cs]: As a point of interest how did the property names end up camelCased?

**Review by ryanbrandenburg — APPROVED** [review]
Looks great! This will be a great starting point for any future tests.

File: src/Middleware/NodeServices/test/NodeServicesTest.cs (hunk 1 above)
Comment (SteveSandersonMS) [reply src/Middleware/NodeServices/test/NodeServicesTest.cs]: It uses Json.NET with this config: https://github.com/aspnet/AspNetCore/blob/master/src/Middleware/NodeServices/src/HostingModels/HttpNodeInstance.cs#L27

**Pull request merged**